rct229/ruletest_engine/ruletest_jsons/ashrae9012019/system_types/System_10_Warm_Air_Furnace_Elec.json

```diff
@@ -34,8 +30,15 @@
                                     "id": "System 10",
                                     "heating_system": {
                                         "id": "Furnace Coil 1",
-                                        "type": "FURNACE",
+                                        "type": "ELECTRIC_RESISTANCE",
                                         "energy_source_type": "ELECTRICITY"
+                                    },
+                                    "fan_system": {
+                                        "id": "Fan System 1",
+                                        "fan_control": "CONSTANT",
+                                        "fan": {
```

Comment:
Was this supposed to be a supply_fans key perhaps? This isn't a valid key via the schema for FanSystem.

Reply:
Yes, silly mistake! I made that update